src/Integration.UnitTests/MefServices/ConfigScopeUpdaterTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
```

Comment:
You desperately need global usings :)
https://github.com/SonarSource/sonar-dotnet/blob/72e14ee23ad494c38c1f6d80c3155991ee21f6eb/analyzers/tests/SonarAnalyzer.Test/SonarAnalyzer.Test.csproj#L76C3-L81C46

What can be added globally, add it to your targets for tests. What can be added only locally to individual CSPRoj, add it only there (maybe Moq is not everywhere, I don't know)

Reply:
..not in this PR

Reply:
I think we have them in newer libraries....